docs/man/docker-pull.1.md

```diff
@@ -19,7 +19,9 @@ It is also possible to specify a non-default registry to pull from.
 
 # OPTIONS
 **-a**, **--all-tags**=*true*|*false*
-   Download all tagged images in the repository. The default is *false*.
+  Download all tagged images in the repository. The default is *false*.
+**-i**, **--insecure-registry**=*true*|*false*
```

Comment:
There really shouldn't be a short flag for insecure. If one wants this, please type it out.